cms/static/js/views/import.js

```diff
@@ -320,7 +342,14 @@ define(
              * @return {JSON} the data of the previous import
              */
             storedImport: function () {
-                return JSON.parse($.cookie(COOKIE_NAME));
+                var import_cookie ;
+                $.each(filter_cookies(COOKIE_NAME), function(key, stored_cookie) {
+                    var cookie_file_url = stored_cookie[COOKIE_NAME].file.url;
+                    if(cookie_file_url.indexOf(course.id) != -1){
```

Comment:
@Shrhawk yeah I guess so. Because `course` object would not be present there. 

So here are two suggestions that I can think of.

1. Add  `library` object just like `course` object to make the codebase consistent.

Or 2. Use some URL manipulation : I am assuming Libraries are always `org+library_code` url structure.
So you can retrieve library_id by some thing like `library_id = document.location.pathname.split('/').slice(-1).pop()`

But I think method 1 is a more better approach.